orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/DeployManifestContext.java

```diff
@@ -17,88 +17,65 @@
-  public DeployManifestContext(
-      @JsonProperty("source") String source,
-      @JsonProperty("manifestArtifactId") String manifestArtifactId,
-      @JsonProperty("manifestArtifact") Artifact manifestArtifact,
-      @JsonProperty("manifestArtifactAccount") String manifestArtifactAccount,
-      @JsonProperty("skipExpressionEvaluation") Boolean skipExpressionEvaluation,
-      @JsonProperty("trafficManagement") TrafficManagement trafficManagement,
-      @JsonProperty("requiredArtifactIds") List<String> requiredArtifactIds,
-      @JsonProperty("requiredArtifacts") List<BindArtifact> requiredArtifacts) {
-    this.source = source;
-    this.manifestArtifactId = manifestArtifactId;
-    this.manifestArtifact = manifestArtifact;
-    this.manifestArtifactAccount = manifestArtifactAccount;
-    this.skipExpressionEvaluation = skipExpressionEvaluation;
-    this.trafficManagement =
-        Optional.ofNullable(trafficManagement).orElse(new TrafficManagement(false, null));
-    this.requiredArtifactIds = requiredArtifactIds;
-    this.requiredArtifacts = requiredArtifacts;
-  }
+  private TrafficManagement trafficManagement;
 
-  @Getter
-  public static class BindArtifact {
-    @Nullable private final String expectedArtifactId;
+  private Source source;
 
-    @Nullable private final Artifact artifact;
+  private String manifestArtifactId;
+  private Artifact manifestArtifact;
+  private String manifestArtifactAccount;
 
-    public BindArtifact(
-        @JsonProperty("expectedArtifactId") @Nullable String expectedArtifactId,
-        @JsonProperty("artifact") @Nullable Artifact artifact) {
-      this.expectedArtifactId = expectedArtifactId;
-      this.artifact = artifact;
-    }
-  }
+  private List<String> requiredArtifactIds;
+  private List<BindArtifact> requiredArtifacts;
 
-  @Getter
+  @Builder.Default private boolean skipExpressionEvaluation = false;
+
+  @Data
   public static class TrafficManagement {
-    private final boolean enabled;
-    private final Options options;
+    private boolean enabled = false;
+    private Options options = new Options();
 
-    public TrafficManagement(
-        @JsonProperty("enabled") Boolean enabled, @JsonProperty("options") Options options) {
-      this.enabled = Optional.ofNullable(enabled).orElse(false);
-      this.options =
-          Optional.ofNullable(options).orElse(new Options(false, Collections.emptyList(), null));
+    @Data
+    public static class Options {
+      private boolean enableTraffic = false;
+      private List<String> services = Collections.emptyList();
+      private ManifestStrategyType strategy = ManifestStrategyType.None;
     }
 
-    @Getter
-    public static class Options {
-      private final boolean enableTraffic;
-      private final List<String> services;
-      private final ManifestStrategyType strategy;
+    public enum ManifestStrategyType {
+      @JsonProperty("redblack")
+      RedBlack,
+
+      @JsonProperty("highlander")
```

Comment:
This changes the serialized form of the `ManifestStrategyType`; prior to this it was capitalized.  This will break deploying in clouddriver, as clouddriver is expecting this to be capitalized as it was prior to the change.

Reply:
good catch.
I've reverted it back to how it was previously.